onnxruntime/core/framework/allocatormgr.h

```diff
@@ -6,11 +6,17 @@
 #include "core/common/common.h"
 #include "core/framework/allocator.h"
 #include "core/session/onnxruntime_c_api.h"
+#include <map>
```

Comment:
>map [](start = 10, length = 3)

any particular reason for map? If not, I'd recommend unordered_map

Reply:
copied from execution_provider.h

---
In reply to: [564296907](https://github.com/microsoft/onnxruntime/pull/6332#discussion_r564296907) [](ancestors = 564296907)

Reply:
will update it.

---
In reply to: [564717612](https://github.com/microsoft/onnxruntime/pull/6332#discussion_r564717612) [](ancestors = 564717612,564296907)